harvard-lts/etd_alma_service
# ETD-332: add timestamp

**add timestamp to export.**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-332

# What does this Pull Request do?
adds timestamp to alma service xml export file

# How should this be tested?
1) download branch
2) get `alma_proquest_id_rsa` &` known_hosts` from vault and put in `.ssh/`
3) copy env-example to .env and set to dev settings, ensure mongo values are set
4) start container `docker-compose -f docker-compose-local.yml up -d --build --force-recreate`
5) `pytest `. All tests should pass
6) `python3 scripts/invoke-task.py`
6) go into the alma dropbox by typing `sftp -i .ssh/alma_dropbox_id_rsa huletd@hegel`.  cd to `incoming` and then do an ls. you should see a file in the format `AlmaDelivery_dev_<DATETIMESTAMP>.xml`
7) shutdown container `docker-compose -f docker-compose-local.yml down`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? y

# Interested parties
Tag (@ mention) interested parties: @awoods
